go/node/docker.go

```diff
@@ -0,0 +1,152 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/obscuronet/go-obscuro/go/common/docker"
+)
+
+type DockerNode struct {
+	cfg *Config
+}
+
+func NewDockerNode(cfg *Config) (Node, error) {
+	return &DockerNode{
+		cfg: cfg,
+	}, nil // todo: add config validation
+}
+
+func (d *DockerNode) Start() error {
+	// TODO this should probably be removed in the future
+
+	err := d.startEdgelessDB()
```

Comment:
what is that TODO saying should be removed? The startEdgelessDB call? I feel like it should probably be startDatabase() or something to be at the same level of abstraction as host and enclave.

Reply:
I added a print there, which for some reason didn't make it. 
It's printing the enclave execution options, helpful for debugging but should be removed.